Automattic/jetpack
# Added password strength checker class and Sync listener.

This adds a class that listens to password verification events and checks the passwords for certain criteria. 

#### Changes proposed in this Pull Request:

* Adds a password checker class.
* Adds a new Sync listener warning for insecure passwords.

#### Testing instructions:
* Use a Jetpack site with this PR.
* Sandbox your connection and apply D25323-code to your sandbox.
* Set an insecure password for your user, for example, the same as your login.
* Observe an audit log entry and a sticker in the MC.

#### Proposed changelog entry for your changes:
* Added a password checker class.
